src/components/Hero.style.js

```diff
@@ -0,0 +1,75 @@
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+export const DivStyled = styled.div`
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+`;
+
+export const H1Styled = styled.h1`
+  font-family: Bitter;
+  font-size: 38px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  text-align: center;
+  color: var(--gray-dark);
+  margin-top: 27px;
+  margin-bottom: 7px;
+}
+`;
+
+export const H2Styled = styled.h2`
+  font-family: Montserrat;
```

Comment:
the font-styles like family, size, weight etc should be defined in [global styles](https://styled-components.com/docs/api#createglobalstyle). They will be used throughout the app and are usually the same for all h1, all h2, etc